src/display/api.js

```diff
@@ -15,6 +15,10 @@
 /* globals requirejs, __non_webpack_require__ */
 /* eslint no-var: error */
 
+/**
+ * @module pdfjs
```

Comment:
Given that the `PDFJS` global was removed, and that we're now using `pdfjsLib` for code running on the main-thread, is that name really appropriate at this point!?

I readily admit not knowing much about JSDoc `@module`, but this was something that stood out to me.

Reply:
I checked how we import `pdfjs-dist` in the examples and we indeed use `pdfjsLib` there, so that sounds like a better name for the documentation too. I have changed this.